File: lab_2/generator.cpp
#include <ctime>
#include <iostream>

#define ROW_SIZE 128


//This file contains function that generates 
//pseudo-random binary row by defined size.
void generate_row() {
    srand(time(NULL));
    for (int i(0); i < ROW_SIZE; ++i) {
        std::cout << rand() % 2;
    }
    std::cout << std::endl;
}

int main() {
    generate_row();
    return 0;
}

File: lab_2/Generator.java
import java.util.Random;


public class Generator{
    final int ROW_SIZE = 128;

    public void generate_row(){
        Random rnd = new Random();
        for (var i = 0; i < ROW_SIZE; ++i){
            var bit = rnd.nextInt(2);
            System.out.print(bit);
        }
    }

    public static void main(String[] args){
        var generator = new Generator();
        generator.generate_row();
    }
}